code/modules/mob/living/status_procs.dm

```diff
@@ -128,28 +128,24 @@
 
 /mob/living/proc/blur_eyes(amount)
 	if(amount>0)
-		var/old_eye_blurry = eye_blurry
 		eye_blurry = max(amount, eye_blurry)
-		if(!old_eye_blurry)
-			overlay_fullscreen("blurry", /obj/screen/fullscreen/blurry)
```

Comment:
If this overlay is not used elsewhere, you should remove it

Reply:
I've already removed it, it was used only here